yunikorn-web.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: yunikorn-web
+  version: 1.6.0
+  epoch: 0
+  description: Apache YuniKorn Web UI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - node-gyp
+      - nodejs-18
+      - npm
+      - pnpm
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: a51cc7203533f4134f3c1ea32a714f33876217f6
+      repository: https://github.com/apache/yunikorn-web
+      tag: v${{package.version}}
+
+  - uses: patch
+    with:
+      patches: |
+        0001-increase-the-budget.patch
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/html
+      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1 pnpm i
+      pnpm build:prod
+      cp -r dist/yunikorn-web/* ${{targets.contextdir}}/html/
```

Comment:
These should live somewhere like `/usr/share/yunikorn-web/html` - then a compat package can make the `${{targets.contextdir}}/html/` symlink to the files